core/src/components/img/img.tsx

```diff
@@ -29,9 +31,12 @@ export class Img implements ComponentInterface {
     this.addIO();
   }
 
-  /** Emitted when the img src is loaded */
+  /** Emitted when the img src is loaded (error included) */
```

Comment:
```suggestion
  /** Emitted when the img src has been set */
```
I realize the `ionImgDidLoad` event is a bit misleading, but for now let's just update the description so it's very clear what the event does.